locals.tf

```diff
@@ -55,9 +55,10 @@ locals {
   # Ensure max builds is optional
   runners_max_builds_string = var.runners_max_builds == 0 ? "" : format("MaxBuilds = %d", var.runners_max_builds)
 
-  # Define key for runner token for SSM
-  secure_parameter_store_runner_token_key  = "${var.environment}-${var.secure_parameter_store_runner_token_key}"
-  secure_parameter_store_runner_sentry_dsn = "${var.environment}-${var.secure_parameter_store_runner_sentry_dsn}"
+  # Define key for runner and registration token for SSM
+  secure_parameter_store_registration_token_key = "${var.environment}-${var.secure_parameter_store_registration_token_key}"
```

Comment:
issue: do not prepend the `environment` here. This is done for the other parameters as they are managed by this module and we have to create unique names. But your new parameter is not managed by this module, but outside (by the caller). Thus we should simply use the name provided by the caller.

Reply:
Make sense to me, will change.